package/Dockerfile

```diff
@@ -8,8 +8,9 @@ RUN apk add -U curl xz
 ENV KUBECTL_VERSION v1.19.7
 RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${ARCH}/kubectl && \
     chmod +x kubectl
+ENV K9S_VERSION v0.24.10
 RUN if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
-    curl -L https://github.com/derailed/k9s/releases/download/v0.20.2/k9s_Linux_${ARCH}.tar.gz | tar xvzf -
+    curl -L https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9s_${K9S_VERSION}_Linux_${ARCH}.tar.gz | tar xvzf -
```

Comment:
The URL changed from release v0.24.9 to v0.24.10 and will switch back to the old version in the next release. See https://github.com/looztra/asdf-k9s/pull/20 and https://github.com/looztra/asdf-k9s/issues/18

Please add a comment to remove the second `${K9S_VERSION}` on the next bump.